comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter_test.go

```diff
@@ -72,17 +72,19 @@ func TestLogsExporter(t *testing.T) {
 					lrr := testutil.GenerateLogsOneLogRecord()
 					ldd := lrr.ResourceLogs().At(0).ScopeLogs().At(0).LogRecords().At(0)
 					ldd.Attributes().PutStr("message", "hello")
+					ldd.Attributes().PutStr("datadog.log.source", "custom_source")
```

Comment:
Can you add another test that sets this attribute in resource attributes?

Reply:
Added in last commit 👍